examples/gno.land/r/demo/boards/post.gno

```diff
@@ -179,7 +199,8 @@ func (post *Post) RenderSummary() string {
 	str += "\\- " + displayAddressMD(post.creator) + ","
 	str += " [" + post.createdAt.Format("2006-01-02 3:04pm MST") + "](" + post.GetURL() + ")"
 	str += " \\[[x](" + post.GetDeleteFormURL() + ")]"
-	str += " (" + strconv.Itoa(post.replies.Size()) + " replies)" + "\n"
+	str += " (" + strconv.Itoa(post.replies.Size()) + " replies)"
+	str += " (" + strconv.Itoa(post.reposts.Size()) + " reposts)" + "\n"
 	return str
```

Comment:
would it not make more sense to have this as `(%d replies, %d reposts)`?

Reply:
I was making minimal changes to existing code, and in the mobile app these are separate displays like on Mastodon:
![Screenshot 2024-01-31 at 13 55 14](https://github.com/gnolang/gno/assets/1999543/e144ddde-43b8-406d-a595-5dee5b698e76)

If you think they should be combined, I'll make the change. What do you think?

Reply:
Maybe not for r/demo/boards, but GnoSocial may eventually have more metrics like number of "likes", number of views, etc. We will have to get feedback on the best display option. Then we can update r/demo/boards with the best option. Maybe we leave this as is for now?